### ISSUE TITLE
Reminder option for calendars

### ISSUE THREAD

**Issue by agageldidurdyyev** [AnandChowdhary/calendar-link#567]
I would like to add this option as well, how can I do it?

```JS
      'reminders': {
        'useDefault': false,
        'overrides': [
          {'method': 'email', 'minutes': 24 * 60},
          {'method': 'popup', 'minutes': 10}
        ]
      }
```

**Comment by AnandChowdhary** [comment]
Is this supported in URLs and ICAL?

**Comment by AnandChowdhary** [comment]
Right, so we know for sure that the Google Calendar API allows for this, and so does ICAL (https://stackoverflow.com/a/15038924/1656944), but since calendar-link is based on URLs, we need to verify whether there's a URL parameter which will set this up. AFAIK, this is unsupported by major calendar providers.

**Comment by agageldidurdyyev** [comment]
Google Calendar and iCal is enough. Could we add this option(I could try to do this as well, can I fork the project and work on it)?

**Comment by AnandChowdhary** [comment]
Go for it, would love to have you contribute. Thanks a ton!

**Comment by FindingAnand** [comment]
⚠️ This issue has not seen any activity in the past 2 months so I'm marking it as stale. I'll close it if it doesn't see any activity in the coming week.

**Comment by agageldidurdyyev** [comment]
Don't close it yet

**Comment by FindingAnand** [comment]
⚠️ This issue has not seen any activity in the past 2 months so I'm marking it as stale. I'll close it if it doesn't see any activity in the coming week.
